README.md

```diff
@@ -19,6 +19,14 @@ $ cd git-machete
 $ sudo make install
 ```
 
+Installation could be done by following commands:
```

Comment:
So since now there's the new way of installation with setup.py, maybe remove the above instruction? It looks confusing at the moment since it's not clear whether the two instructions are alternatives to each other or they should be performed one after another etc.

Reply:
I will clarify it, thx for tip.